landoapi/hgexportbuilder.py

```diff
@@ -0,0 +1,32 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""
+Module for constructing Mercurial patches in 'hg export' format.
+"""
+
+HG_EXPORT_PATCH_TEMPLATE = """# HG changeset patch
+# User {author}
+# Date {patchdate}
+{commitdesc}
```

Comment:
There is much more that will have to go into getting commit description. It needs the following format:

Bug #: {Revision Title} r={reviewers}

{Revision Summary}
{Phabricator Revision URL ?}